tests/test.html

```diff
@@ -75,19 +75,37 @@
 
         <p>
             <strong>clndr.constraints</strong> Test start and end constraints.
-            (4th of this month to the 12th of next month). Logs in the console.
+            (the 4th of this month to the 12th of next month). Logs in the console.
         </p>
         <div id="constraints" class="cal1"></div>
 
         <p>
+            <strong>clndr.prevNextMonthConstriants</strong> Test start and end constraints.
```

Comment:
Sorry to nitpick :grin: some of these are misspelled `constriant` and I don't want anyone to get confused if they copy & paste them!